.travis.yml

```diff
@@ -5,6 +5,7 @@ sudo: false
 stages:
   - checks
   - tests
+  - optional
```

Comment:
Move the branch history check as a third stage, see https://github.com/mfem/mfem/pull/1229#discussion_r368207023.

I don't think it is easy to distinguish potential legitimate changes from potential bad commits.